.devcontainer/Dockerfile

```diff
@@ -9,4 +9,5 @@ RUN apt-get install -y \
         unzip \
         git \
         m4 \
-        libyaml-dev
+        libyaml-dev \
+        libcurl4-openssl-dev
```

Comment:
I think we don't need to introduce a dependence for the release if it's only used in tests.
I'm not familiar with the build process and this looks like a part of it to me.

Reply:
Hey, thanks for the review!
This Dockerfile is not used for releases though! It is the `.devcontainer`/Dockerfile. It's used for development: https://code.visualstudio.com/docs/devcontainers/containers
For me it was the only way I could get the kong stack running locally to run the integration tests on my own PC.

So it's not part of the kong build process.